packages/functional-tests/pages/relier.ts

```diff
@@ -5,7 +5,7 @@ export class RelierPage extends BaseLayout {
     const url = query
       ? `${this.target.relierUrl}?${query}`
       : this.target.relierUrl;
-    return this.page.goto(url, { waitUntil: 'networkidle' });
+    return this.page.goto(url, { waitUntil: 'load' });
```

Comment:
I see the [playwright docs](https://playwright.dev/docs/next/api/class-frame#frame-goto-option-wait-until) say, "rely on web assertions to assess readiness instead". Instead of `waitUntil: 'load'`, should we be asserting that an expected element is present? I don't see that the docs discourage `'load'` and that certainly makes it easier to clean up, but I'm not sure if the web assertion would be considered less flaky than this. 🤔

Reply:
Yes we have used `expect` wherever we could use web assertions, but for `goto()` I felt it's better to use `load` to fully redirect and load the concern page before performing any actions. Hence for all the `goto()` I have used an additional `load` parameter.